frontend/packages/knative-plugin/src/models.ts

```diff
@@ -98,3 +98,16 @@ export const EventSourceContainerModel: K8sKind = {
   namespaced: true,
   crd: true,
 };
+
+export const EventSourceApiServerModel: K8sKind = {
+  apiGroup: 'sources.eventing.knative.dev',
+  apiVersion: 'v1alpha1',
+  kind: 'ApiServerSource',
+  label: 'ApiServerSource',
+  labelPlural: 'ApiServerSources',
+  plural: 'ApiServerSources',
```

Comment:
Not sure how important it is but this doesn't match what is in the other event sources. The others are all lower and this is camel case.